tests/chainermn_tests/communicator_tests/test_mpi.py

```diff
@@ -0,0 +1,104 @@
+import os
+try:
+    import queue
+except ImportError:
+    import Queue as queue
+import re
+import signal
+from subprocess import CalledProcessError
+from subprocess import PIPE
+from subprocess import Popen
+import sys
+import threading
+import unittest
+
+import numpy as np
+import pytest
+
+import chainer.testing
+import chainer.testing.attr
+import chainermn
+from chainermn.communicators import _memory_utility
+
+
+class _TimeoutThread(threading.Thread):
+    def __init__(self, queue, rank):
+        super(_TimeoutThread, self).__init__()
+        self.queue = queue
+        self.rank = rank
+
+    def run(self):
+        try:
+            self.queue.get(timeout=60)
+        except queue.Empty:
+            # Show error message and information of the problem
+            try:
+                p = Popen(['ompi_info', '--all', '--parsable'], stdout=PIPE)
+                out, err = p.communicate()
+                if type(out) == bytes:
+                    out = out.decode('utf-8')
+                m = re.search(r'ompi:version:full:(\S+)', out)
+                version = m.group(1)
+
+                msg = "\n\n***** ERROR:" \
+                      "It looks like you are using " \
+                      "Open MPI version {}." \
```

Comment:
Can you add a line break here?

Reply:
Improved the error message so the message would be:
```
***** ERROR: It looks like you are using Open MPI version 3.0.0.
***** It is known that the following Open MPI versions have a bug
***** that cause MPI_Bcast() deadlock when GPUDirect is used:
*****  3.0.0, 3.0.1, 3.0.2, 3.1.0, 3.1.1, 3.1.2
```